src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Color color = Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
let's not create Random class instance each time the method is called. So, let's initialize your random object in the class field or using a constructor